src/content/components/youtube/youtube.util.ts

```diff
@@ -12,7 +12,8 @@ export const getYoutubeAcc = (href: string) => {
 		pathname.startsWith('/user/') ||
 		pathname.startsWith('/@')
 	) {
-		validUrl = pathname.split('/').filter(Boolean)[0];
+		const keyWords = ['channel', 'c', 'user'];
+		validUrl = pathname.split('/').filter((el) => el && !keyWords.includes(el))[0];
```

Comment:
I recommend converting path to lowercase before checking

Reply:
updated with your recommendations